rele/client.py

```diff
@@ -57,20 +63,31 @@ def create_subscription(self, subscription, topic):
         :param subscription: str Subscription name
         :param topic: str Topic name to subscribe
         """
-        subscription_path = self._client.subscription_path(
+        subscription_path = self._subscriber_client.subscription_path(
             self._gc_project_id, subscription
         )
-        topic_path = self._client.topic_path(self._gc_project_id, topic)
+        topic_path = self._subscriber_client.topic_path(self._gc_project_id, topic)
 
         with suppress(exceptions.AlreadyExists):
             try:
-                self._client.create_subscription(
-                    name=subscription_path,
-                    topic=topic_path,
-                    ack_deadline_seconds=self._ack_deadline,
-                )
+                self._create_subscription(subscription_path, topic_path)
             except exceptions.NotFound:
-                logger.error("Cannot subscribe to a topic that does not exist.")
+                logger.info(
```

Comment:
I would log it as warning, not info. And I agree that having the topic name would be very helpful

Reply:
Done.